### PR TITLE
EVA now has EVA Suits

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
This adds the EVA suit and helmet and maps them into EVA. There's no crates yet because I want to see if the recent gun PR leads to an epidemic of Cargonia revolts before I give them access to these coveted items.
**Screenshots**

eva screenshot

**Changelog**

:cl: Rane
- add: Added EVA suits to the EVA room.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml [hunk 1]
```diff
@@ -34,6 +34,18 @@
...
+  parent: ClothingHeadBase
+  id: ClothingHeadHelmetEVA
+  name: EVA helmet
+  description: A basic helmet designed for extravehicular activies.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/eva.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/eva.rsi
+  - type: IngestionBlocker
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Head/helmets.yml:47]: no pressure protection on the helmet?

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Clothing/Head/helmets.yml:47]: None of the helmets currently have pressure protection and it works fine without them. Look at the cosmonaut helmet.

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Clothing/Head/helmets.yml:47]: I added pressure to the helmet and now you can spacewalk with just the helmet and take no pressure damage.

**Review by PaulRitter — APPROVED** [review]
we need to change pressure protection but for now remove it from the helmet

**Pull request merged**